blockstore/src/test/java/brooklyn/location/blockstore/openstack/RackspaceVolumeManagerLiveTest.java

```diff
@@ -44,6 +44,11 @@ protected String getDefaultAvailabilityZone() {
     }
 
     @Override
+    protected char getDefaultDeviseSuffix() {
+        return 0;
```

Comment:
Do you think it's better to `throw new IllegalStateException("figure out the correct suffix")` instead?